Dockerfile

```diff
@@ -15,31 +11,20 @@ RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-k
 # We add postgis as well to prevent build errors (that we dont see on local builds)
 # on docker hub e.g.
 # The following packages have unmet dependencies:
-RUN apt-get -y update; apt-get install -y postgresql-client-9.6 postgresql-common postgresql-9.6 postgresql-9.6-postgis-2.4 postgresql-9.6-pgrouting netcat
+RUN apt -y update; apt install -y postgresql-client-9.6 postgresql-common postgresql-9.6 postgresql-9.6-postgis-2.4 postgresql-9.6-pgrouting netcat
```

Comment:
Currently, the docker image lacks shp2pgsql and ogr2ogr. Should we not install gdal-bin and PostGIS since postgresql-9.6-postgis-2.4 does not install shp2pgsql. This can be referenced https://github.com/kartoza/docker-osm/issues/49

Reply:
if you need these you should put them on your client system Convince me that they need to be in the docker image which in most cases will be pure server side